common/beerocks/bcl/source/beerocks_socket_thread.cpp

```diff
@@ -349,6 +349,6 @@ bool socket_thread::work()
         }
         // The loop should go over all the sockets. In case something break the for loop before it ended,
```

Comment:
in commit message - type:
`This commit ands a check`
should be
`This commit adds a check`
(nitpick)